### PR TITLE
Adding DatePicker component.

### PR DESCRIPTION
**Pull request by psachs21**

### CODE REVIEW

File: infl-styles/table.css [hunk 1]
```diff
@@ -16,6 +16,5 @@
   display: -webkit-flex;
   display: -ms-flexbox;
   display: flex;
-  overflow: hidden;
```
Comment (SeeThruHead) [inline infl-styles/table.css]: this is datepicker related?

Comment (psachs21) [inline infl-styles/table.css]: Woops, nope. I'll remove these.

File: package.json [hunk 2]
```diff
@@ -126,6 +126,7 @@
     "jss-nested": "^1.0.1",
     "lodash": "^3.7.0",
     "lost": "^6.7.1",
+    "moment": "^2.11.2",
```
Comment (SeeThruHead) [inline package.json]: moment is fairly large if you include all of it's translations

Comment (psachs21) [inline package.json]: I don't think this package is all the translations. For that, you have to include moment-locales I believe.

File: src/date-picker/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react';
+import DateTime from 'react-datetime';
+
+// This component really just extends react-datetime
+// found at https://github.com/arqex/react-datetime.
+
+
+// import './date-picker.scss';
```
Comment (SeeThruHead) [inline src/date-picker/index.js]: you can wrap it in a div, and modulize the wrapper class -- see patternity-evo/src/react-select

Comment (psachs21) [inline src/date-picker/index.js]: Ah good point. I'll do that.

**Pull request merged**